documents/data-sharing-annual-community-survey.md

```diff
@@ -0,0 +1,87 @@
+# Annual community survey data sharing
+
+This document describes how data from the annual survey should be shared.
+
+## Objectives
+
+In priority order:
+
+* Protect the privacy of respondents.
+* Provide high quality data (unbiased data which is unlikely to be misinterpreted, c.f., raw data).
+* Provide useful and actionable data to the Rust Project (i.e., teams and working groups, contributors) and Foundation.
+* Provide data in a timely manner.
+* Provide data which is generally useful for the community (users and potential users of Rust, and providers of Rust tools and services). Note that this does not mean producing data to be useful for specific segments of the community.
+* Provide marketing for Rust.
+* Avoid overloading people working directly with the survey data (note that in 2021 there were over 9000 completed surveys and multiple opportunities for free form responses).
```

Comment:
Nitpick: I'm glad "avoid overloading people working directly with the survey data" is listed here. I suspect it is not prioritized highly enough.

E.g. I think it is higher priority than providing marketing for Rust. It may even be higher priority than "Provide data that is generally useful to the community."